dev-python/kaitaistruct/kaitaistruct-0.8.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
How did you test it?

Reply:
Only via mitmproxy working.  I didn't see anything else that depends on it, if there is a more through way let me know.

Reply:
upstream is at 3.7

Reply:
Dropping 3.8